Refresh quoted by messages when quoted message changes (#3553)
GetStream/stream-chat-swift: CHANGELOG.md

- Add `ImageAttachmentPayload.file` for setting `file_size` and `mime_type` for image attachments [#3548](https://github.com/GetStream/stream-chat-swift/pull/3548)
### 🐞 Fixed
- Remove the main thread requirement from the `DataStore` [#3541](https://github.com/GetStream/stream-chat-swift/pull/3541)

> - Quoted message preview did not update when the quoted message was deleted [#3553](https://github.com/GetStream/stream-chat-swift/pull/3553)

Reviewer: Usually, changelogs are written in the present, not in the past: 
```suggestion
- Fix quoted message preview not updating when the quoted message is deleted [#3553](https://github.com/GetStream/stream-chat-swift/pull/3553)
```
